ui/src/App.js

```diff
@@ -86,12 +89,20 @@ const Admin = (props) => {
     >
       {(permissions) => [
         <Resource name="album" {...album} options={{ subMenu: 'albumList' }} />,
-        <Resource name="artist" {...artist} options={{ subMenu: 'library' }} />,
-        <Resource name="song" {...song} options={{ subMenu: 'library' }} />,
+        <Resource
+          name="artist"
+          {...artist}
+          options={{ subMenu: 'library', onActive: MicIcon }}
```

Comment:
I don't like the idea of having the _inactive_ icon in one place (in the `artist/index.js`) and the _active_ icon in another (here). Also, IMO `onActive` is an event name, not an icon/image. Maybe rename this to `activeIcon`?

What do you think about  creating an `DynamicIcon` component, that receives the two icons and the resource name, and it then renders the appropriate icon?

Reply:
Yeah sure @deluan  i will make a DynamicIcon component.